src/codewhisperer/models/constants.ts

```diff
@@ -6,9 +6,9 @@
 /**
  * SDK Client
  */
-export const endpoint = 'https://codewhisperer.us-east-1.amazonaws.com/'
+export const endpoint = process.env.CODEWHISPERER_ENDPOINT ?? 'https://codewhisperer.us-east-1.amazonaws.com/'
 
-export const region = 'us-east-1'
+export const region = process.env.CODEWHISPERER_REGION ?? 'us-east-1'
```

Comment:
Should be documented in https://github.com/aws/aws-toolkit-vscode/blob/master/CONTRIBUTING.md#service-endpoints

off-topic: we really should have all env vars in one place. Otherwise it's not clear at all where random env vars can change the behavior anywhere in the codebase.

Reply:
This does mean customers can change them too, is this expected?

Reply:
I think the better way is to follow what @hayemaxi suggests to make it a dev setting. 

> This does mean customers can change them too, is this expected?

IMO CX can make these changes anyway by editing the src code and build plugins on their own, thus I don't think it's a concern to make this change?

Reply:
that's fair.